src/content/self-hosted/aks.mdx

```diff
@@ -41,6 +41,8 @@ Run the following command to obtain your cluster's API server endpoint:
 kubectl config view --minify | grep server
 ```
 
+> Our installation guides assume Okteto will be running in a dedicated cluster. If you want to install Okteto in a cluster with other workloads, we recomment to contact support for extra guidance.
```

Comment:
@rberrelleza what would be the right CTA here?

Reply:
I think talk to us is the right CTA, but let's rephrase it. @bigash42 could your team help here?